docs/tips/comparison.md

```diff
@@ -0,0 +1,32 @@
+# Envd vs. Other tools
```

Comment:
Should all `envd` be in code format and small case?

Reply:
To align with other pages, "envd" should be in the code format, quoted by "`". Can check the source code of other pages.

Reply:
Added "`"